modules/devguide/examples/go/transactions.go

```diff
@@ -0,0 +1,812 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/couchbase/gocb/v2"
+)
+
+func main() {
```

Comment:
Noticed we're not calling any of the functions in `main()` - just double checking this is intentional because the example is not intended to be runnable?

Reply:
I was focusing on getting the docs and examples written. It should all the runnable though.